src/value.rs

```diff
@@ -89,6 +93,9 @@ use gstring::GString;
 use translate::*;
 use types::{StaticType, Type};
 
+#[derive(Debug, PartialEq)]
+pub struct ValueTypeMismatch;
```

Comment:
Should impl `std::error::Error`

Reply:
Maybe should also carry the two `glib::Type`s, the expected and the one we found? That would allow for a more useful `Display` impl at least.

Reply:
Yes! I was doing exactlty so :) Thanks for the heads-up